src/main/java/eightbit/command/EventCommand.java

```diff
@@ -0,0 +1,37 @@
+package eightbit.command;
+
+import eightbit.task.Event;
+import eightbit.util.Storage;
+import eightbit.util.TaskList;
+
+/**
+ * Represents a command to add an event.
+ */
+public class EventCommand extends Command {
+
+    private final Event event;
+
+    /**
+     * Constructs a command to add an event.
+     *
+     * @param event Event instance to be added.
+     */
+    public EventCommand(Event event) {
+        this.event = event;
+    }
+
+    /**
+     * Adds the event into the user's list.
+     *
+     * @param taskList User's list of tasks.
+     * @param storage  Storage responsible for reading/writing the file.
+     * @return The response after executing the command.
+     */
+    @Override
+    public String execute(TaskList taskList, Storage storage) {
```

Comment:
I think it would be better to name the method after the functionality, instead of execute() which may not be the most intuitive. Perhaps something like addEventToList()?